packages/core/src/utils/createMap/pullPolarStyleToShadow.ts

```diff
@@ -0,0 +1,39 @@
+const defaultStylePath = './style.css'
+
+export const pullPolarStyleToShadow = (
+  shadowRoot: ShadowRoot,
+  stylePath = defaultStylePath
+) => {
+  // @ts-expect-error | 'TS2339: Property 'env' does not exist on type 'ImportMeta'.' - It does since we're using vite as a bundler.
+  const devMode = import.meta.env.DEV
+  // move polar css to Shadow DOM (customer has to import it)
+  const attributeName = devMode ? 'style' : 'link'
+  const stylesheets = [...document.getElementsByTagName(attributeName)]
+  const stylesheetDataAttribute = devMode ? 'data-vite-dev-id' : 'data-polar'
+  const polarStylesheets = stylesheets.filter((el) =>
+    el.getAttribute(stylesheetDataAttribute)
+  )
+  if (polarStylesheets.length > 0) {
+    polarStylesheets.forEach((style) => shadowRoot.appendChild(style))
```

Comment:
The nodes are no longer cloned, but moved. We previously left the imported CSS in the root document, assuming there would be no side-effects since we never noticed any. However, they exist, and classnames like `.row` and `.col-xx` are set.

The fontawesome styles actually required in the root due to mentioned browser bug are now handled separately.